src/components/counter.js

```diff
@@ -0,0 +1,90 @@
+// @ts-check
+
+import React, { Component } from 'react'; 
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import {StyleSheet, Text, View} from 'react-native'
+import { Button } from 'react-native-elements'
+
+import {counterIncrement, counterDecrement}  from '../actions/counter';
+
+const styles = StyleSheet.create({ 
+    container: { 
+        flex: 1, 
+        padding: 30, 
+        justifyContent: 'center', 
+        alignItems: 'center', 
+        backgroundColor: '#000080',
+    },
+
+    counter: {
+        padding: 30,
+        alignSelf: 'center',
+        fontSize: 26,
+        fontWeight: 'bold',
+    },
+    text: {
+        fontFamily: "Roboto-Light",
+        fontSize: 40,
+        textAlign: "center",
+        margin: 10
+    },
+});
+
+class Learning_Project extends Component { 
+    render() {
+        return( 
+            <View style={{
+                flex: 1,
+                padding: 20,
+                flexDirection: 'row',
+                justifyContent: 'center',
+              }}>
+            <Button
+            large
+            title='Increment +'
+            color = "green"
+            //icon={{name: 'envira', type: 'font-awesome'}}
+            onPress={this.props.actions.counterIncrement}/>
+            
+            <Text>
+                <Text style={styles.text}>
+                {this.props.count}
+                </Text>
+            </Text>
+
+             <Button
+            large
+            title='Decrement -'
+            color = "red"
+            
+            onPress={this.props.actions.counterDecrement}
+             />
+            </View>
+            );
+            
+                           
+            //     title=" Increment + "
```

Comment:
Don't leave commented out code in the codebase unless there is a very good reason to do so. If you need these changes for latter then stash them locally.